### ISSUE TITLE
Topics resources over 500

### ISSUE THREAD

**Issue by dawidor** [Mongey/terraform-provider-kafka#122]
there is an issue with kafka topic deployment when there is more than 500 topics resources to refresh/apply.

Terraform just hangs if there is more than 500 resources to deploy...

Similar fix for ACL:
https://github.com/Mongey/terraform-provider-kafka/commit/674ef007718a7f310f2b17b79169d789fdee4c53

is it possible to have similar for topics? 
https://github.com/Mongey/terraform-provider-kafka/blob/master/kafka/client.go#L215

thanks
R

**Comment by Mongey** [comment]
👋 Thanks for the report. I'll take a look into if we can move that metadata refresh higher in the provider, so that we're not calling it for every topic.

**Comment by dawidor** [comment]
I think I manage to find problem.

So scenario is when you have more than magic ~500 kafka_topic resources already deployed and present terraform.tfstate (centos and rhel verified)
then when you try to  apply any changes then terraform refresh starts hanging on ~500 topic resource. 

I have done some debuging and found that :
1) it stops working on the following line:
https://github.com/Mongey/terraform-provider-kafka/blob/master/kafka/client.go#L217
2) which then with debuging sarama it hangs on
https://github.com/Shopify/sarama/blob/v1.24.1/client.go#L775

So basically it fails on 
Logger.Printf("client/metadata fetching metadata for all topics from broker %s\n", **broker.addr**)

When you remove "broker.addr" from display all works as expected.

**So it looks like the following line is not working and cause the issue:**
https://github.com/Mongey/terraform-provider-kafka/blob/master/kafka/client.go#L27

If I remove it from code then all works as expected. No terraform hangs.

Can you please review it ?

Regards,

Rafal

**Comment by Mongey** [comment]
thanks @dawidor.  I was finally able to reproduce. I removed the logging for Sarama in the 0.2.9. not sure why that causes the issue, but I'll investigate before re-introducing it.

**Comment by dawidor** [comment]
@Mongey all works now.
thanks for the fix.
